sql_user_check_postgres/README.md

```diff
@@ -0,0 +1,21 @@
+database_stats.sh script can be used to test connectivity with Postgres database as well as fetch some stats and metadata. 
+The script supports 2 modes which can be specified with -m flag while running the script. One mode is stats mode which 
+will just print info like list of databases with their size. Count of tables, rows, column, size per schema. 
+Distribution of data by data type. Stats mode is the default mode. Another mode is full_metadata which will print 
+complete information of all columns like their data type, constraints etc.
+
+From sql_user_check_postgres directory 
+
+stats Mode:
+```sql
+./database_stats.sh -u <DATABASE_USER> -h <DATABASE_HOSTNAME> -d <DATABASE_NAME> 
+```
+
+full_metadata Mode:
+```sql
+./database_stats.sh -u <DATABASE_USER> -h <DATABASE_HOSTNAME> -d <DATABASE_NAME> -m full_metadata
+```
+The script will output details in this file ~/test_database_stats.out
```

Comment:
Support a flag called `-o` for file path to write output instead of hard-coded path

Reply:
done